app/grandchallenge/challenges/models.py

```diff
@@ -583,6 +584,17 @@ def status(self):
             status = StatusChoices.CLOSED
         return status
 
+    @property
+    def type(self):
```

Comment:
Don't use `type` for a property as this is confusing with the `type()` built in.